pkg/mesh_install/installer.go

```diff
@@ -246,3 +244,84 @@ func injectGeneratedCertificates(secret *corev1.Secret, cs *cfsslsrv.CFSSLServer
 
 	return secret, nil
 }
+func (i *Installer) reconcileSidecarListForRedisIngress(mesh *v1alpha1.Mesh) {
+	var redisListener json.RawMessage
+	var tempOperatorCUE cuemodule.OperatorCUE
+	var err error
+ReconciliationLoop:
```

Comment:
Just a style point I generally see these in all caps:

```suggestion
RECONCILATION_LOOP:
```

Reply:
I think GoLand gets mad at caps for this. Which is reason enough for me to not care.

Reply:
Ah okay I wasn't aware. It was just a stylistic thing so I'm not hard pressed on it